test/checks/keyboard/landmark-is-top-level.js

```diff
@@ -23,6 +23,16 @@ describe('landmark-is-top-level', function() {
 		assert.deepEqual(checkContext._data, { role: 'main' });
 	});
 
+	it('should return false if the complementary landmark is in another landmark', function() {
+		var mainLandmark = document.createElement('main');
+		var aside = document.createElement('div');
+		aside.setAttribute('role', 'complementary');
+		aside.appendChild(mainLandmark);
+		fixture.appendChild(aside);
```

Comment:
Nitpick: I know, there are multiple tests written with DOM api like above using `createElement`, `appendChild` etc. I found that using the `testutils.fixtureSetup()` function to be helpful for this, and saves us a few lines and also makes it easy to read.

Reply:
I've updated the tests, you're right that they were more difficult to read!